cwm_worker_operator/deployer.py

```diff
@@ -89,6 +89,8 @@ def deploy_worker(domains_config, deployer_metrics, deployments_manager, worker_
             'CDN_CACHE_NOCACHE_REGEX': '\\.({})$'.format('|'.join(volume_config.cache_exclude_extensions)) if len(volume_config.cache_exclude_extensions) > 0 else '',
             'CDN_CACHE_PROXY_CACHE_VALID_200': '{}m'.format(volume_config.cache_expiry_minutes),
             'CDN_CACHE_PROXY_INACTIVE': '{}m'.format(volume_config.cache_expiry_minutes + 1),
+            'DISABLE_HTTP': 'true' if 'http' not in volume_config.protocols_enabled else 'false',
+            'DISABLE_HTTPS': 'true' if 'https' not in volume_config.protocols_enabled else 'false',
```

Comment:
looks like the relevant helm values are boolean and named differently

https://github.com/CloudWebManage/cwm-worker-deployment-minio/blob/main/helm/values.yaml#L356

Reply:
Yes.
That was according to the values mentioned in the issue.
Are you testing that PR?
I'll update the values if you're done with it.
Let me know.

Reply:
I'm not testing it

Reply:
This one? You assigned this to yourself for testing.
https://github.com/CloudWebManage/cwm-worker-cluster/issues/125

Reply:
yes, I will test that one but you should make sure correct values are used